draft-ietf-rtcweb-jsep.xml

```diff
@@ -1935,7 +1935,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         field SHOULD be "-". The value of the
                         &lt;sess-id&gt; field SHOULD be a
                         cryptographically random number. To ensure
-                        uniqueness, this number SHOULD be at least 64
+                        uniqueness, this number SHOULD be 63
```

Comment:
I would write this as "this number should be a 64-bit integer with the bottom 63 bits randomly generated and the top bit set to zero"